tests/testthat/test-mock2-helpers.R

```diff
@@ -0,0 +1,40 @@
+test_that("mock_output_sequence() works", {
+  mocked_sequence <- mock_output_sequence("3", "This is a note", "n")
+  expect_equal(mocked_sequence(), "3")
+  expect_equal(mocked_sequence(), "This is a note")
+  expect_equal(mocked_sequence(), "n")
+  expect_snapshot(mocked_sequence(), error = TRUE)
+})
+
+test_that("mock_output_sequence() works -- force", {
```

Comment:
I tried to write a test that'd fail if I didn't use `force()` but it doesn't fail so I removed `force()`. Maybe because of `list2()`? Or maybe because I missed something obvious. :sweat_smile:

Reply:
Yes, it's no longer needed because `list2` forces all the values in `...`

Reply:
So you can remove this test IMO

Reply:
Ok, done